common/include/lidar_protocol.h

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
IMHO this file could be removed, we are not using it. Please remove files which are not referenced in a commit.